greenriver/hmis-warehouse
# 185646926 auto exit on refer out

## Description
https://www.pivotaltracker.com/story/show/185646926

Auto-exit referrals

Questions: 
* do we care if the referring project is CE or not when auto exiting?
* I'm probably not creating the exit record correctly, please check my implementation 

testing denial:
- create referral from project A to project B
- mark referral as denied pending in project B
- in Admin referrals, approve the denial
- visit project A enrollments, uncheck default filters to show exited enrollments confirm enrollment is exited

testing accepted
- create referral from project A to project B
- mark referral as accepted pending in project B
- complete intake for the newly created enrollment assessment in project B
- visit project A enrollments, uncheck default filters to show exited enrollments confirm enrollment is exited

## Type of change

## Checklist before requesting review
